cmake/EponaConfig.cmake

```diff
@@ -22,8 +22,14 @@ if (UNIX)
     set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -s -O3")
     set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} -g3 -ggdb3 -O0")
 elseif(WIN32)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /MP /W3")
-    add_definitions(-D_CRT_SECURE_NO_WARNINGS)
+	if (CMAKE_CXX_COMPILER_ID MATCHES "GNU")
```

Comment:
It looks like tabulation is mixed, or there are just more whitespaces. Please use 4 whitespaces.

Reply:
fixed